scripts/fix_iss_ebi-ait_hca-ebi-dev-team_389/transfer.py

```diff
@@ -0,0 +1,149 @@
+                    'day': start_date.day,
+                    'month': start_date.month,
+                    'year': start_date.year
+                }
+            },
+            'transferSpec': {
+                'awsS3DataSource': {
+                    'bucketName': source_s3_bucket,
+                    'path': source_s3_path,
+                    'awsAccessKey': {
+                        'accessKeyId': self.access_key_id,
+                        'secretAccessKey': self.secret_access_key
+                    }
+                },
+                'gcsDataSink': {
+                    'bucketName': dest_gs_bucket,
+                    'path': dest_gs_path
+                },
+                'objectConditions': {
+                    'includePrefixes': [source_s3_prefix]
+                }
+            }
+
+        }
+
+        return transfer_job
+
+    def get_operation(self, project_id, job_name):
+        result = self.client.transferOperations().list(name="transferOperations",
+                                                       filter=json.dumps({
+                                                           "project_id": project_id,
+                                                           "job_names": [job_name]
+                                                       })).execute()
+        print('Returned transferOperations: {}'.format(
+            json.dumps(result, indent=4)))
+
+        operations = result.get('operations', [])
+        return operations[0] if len(operations) > 0 else None
+
+    def is_job_done(self, project_id, job_name):
+        operation = self.get_operation(project_id, job_name)
+
+        return operation and operation.get('done', False)
+
+
+def run(command: str, input: str = None, verbose: bool = True):
+    parsed_command = command.split(' ')
+    proc = subprocess.Popen(parsed_command,
+                            stdin=subprocess.PIPE,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE,
+                            )
+    if input:
+        stdout, stderr = proc.communicate(input.encode())
+    else:
+        stdout, stderr = proc.communicate()
+
+    if verbose:
+        print('$ ' + command)
+        print(stdout.decode())
+        print(stderr.decode())
+
+    return proc.returncode, stdout.decode(), stderr.decode()
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument('service_account_credentials_path', help='The file path of GCP Service Account credentials')
+    parser.add_argument('access_key_id', help='Your AWS access key id.')
+    parser.add_argument(
+        'secret_access_key',
+        help='Your AWS secret access key.'
+    )
+
+    args = parser.parse_args()
+
+    description = 'Fix issue: https://github.com/ebi-ait/hca-ebi-dev-team/issues/389'
+    project_id = 'mystical-slate-284720'
```

Comment:
nitpick: would it be useful for future runs to keep this configurable like the aws creds?

Reply:
Yeah, I haven't made this script reusable. It just does what it needs to do to fix the issue.